spec/requests/properties_controller_spec.rb

```diff
@@ -61,6 +61,129 @@
           expect(response).to have_http_status(:unprocessable_entity)
         end
       end
+
+      context "with missing main_home attribute" do
+        let(:request_payload) { { properties: {} } }
+
+        it "returns expected error response", :show_in_doc do
```

Comment:
:show_in_doc is an api pie helper i believe

Reply:
removed